docs/Gemfile

```diff
@@ -17,6 +17,7 @@ gem "jekyll", "~> 4.0.1"
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
   gem "jekyll-sitemap"
+  gem "jekyll-redirect-from"
```

Comment:
We may need to update the Gem lock file and make a copy of it to `dev` branch. This way Snyk and Dependabot work with the latest version.